x11-wm/mutter/mutter-9999.ebuild

```diff
@@ -78,46 +78,43 @@ RDEPEND="
 		>=dev-libs/libgudev-238
 	)
 	systemd? ( sys-apps/systemd )
-	x11-libs/libSM
 	input_devices_wacom? ( >=dev-libs/libwacom-0.13:= )
-	>=x11-libs/startup-notification-0.7
 	screencast? ( >=media-video/pipewire-1.2.0:= )
 	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
 	test? (
 		>=x11-libs/gtk+-3.19.8:3[X,introspection?]
-		gnome-extra/zenity
 	)
 	sysprof? ( >=dev-util/sysprof-capture-3.40.1:4 >=dev-util/sysprof-3.46.0 )
 "
 
-X_OR_XWAYLAND_DEPS="
+X11_CLIENT_DEPS="
 	>=gui-libs/gtk-4.0.0:4[X,introspection?]
+	media-libs/libglvnd[X]
 	>=x11-libs/libX11-1.7.0
 	>=x11-libs/libXcomposite-0.4
 	x11-libs/libXcursor
 	x11-libs/libXdamage
 	x11-libs/libXext
 	>=x11-libs/libXfixes-6
 	>=x11-libs/libXi-1.7.4
-	x11-libs/libxkbfile
```

Comment:
`src/wayland/meta-xwayland.c` includes files from this package so it should stay here afaict.

Reply:
Sorry, I double checked and indeed, it was not an xkbfile include, it was another X11/extensions. My grep was not specific enough.